src/sentry/api/helpers/group_index/update.py

```diff
@@ -255,6 +264,13 @@ def update_groups(
     res_type = None
     activity_type = None
     activity_data: MutableMapping[str, Any | None] | None = None
+    if hasPriority and "priority" in result:
+        handle_priority(
+            priority=result["priority"],
+            reason=result.get("reason", None),
```

Comment:
Doesn't look like the validator has `reason` as a property? I think we can just remove this argument from `handle_priority` since there's no need for a reason here

Reply:
@malwilley Would we want to add the reason to the serializer? For future updates you can also update the reason.

Reply:
I mention this because we have a priority field in the update function already
https://github.com/getsentry/sentry/blob/b3969764e3612e40faf30a445efa6232d27ca898/src/sentry/issues/priority.py#L56-L75

Reply:
No need to add to the serializer, the reason here is just for the activity entry (when auto-updating by Sentry)